apps/web/pages/settings/platform/index.tsx

```diff
@@ -32,7 +32,9 @@ export default function Platform() {
     data: managedUserData,
     refetch: refetchManagedUsers,
   } = useGetOAuthClientManagedUsers(initialClientId);
-  const { data: userBillingData } = useCheckTeamBilling(user?.organizationId);
+  const { data: userBillingData, isFetching: isUserBillingDataLoading } = useCheckTeamBilling(
```

Comment:
I think we can use useGetUserAttributes here as well, if we remove the previous useMe and return isFetching in useGetUserAttributes

Reply:
Indeed